scipy/stats/tests/test_quantile.py

```diff
@@ -95,21 +103,47 @@ def test_against_numpy(self, method, shape_x, shape_p, axis, xp):
 
     @pytest.mark.parametrize('axis', [0, 1])
     @pytest.mark.parametrize('keepdims', [False, True])
-    @pytest.mark.parametrize('nan_policy', ['omit', 'propagate'])
+    @pytest.mark.parametrize('nan_policy', ['omit', 'propagate', 'marray'])
     @pytest.mark.parametrize('dtype', ['float32', 'float64'])
-    def test_against_reference(self, axis, keepdims, nan_policy, dtype, xp):
+    @pytest.mark.parametrize('method', ['linear', 'harrell-davis'])
+    def test_against_reference(self, axis, keepdims, nan_policy, dtype, method, xp):
```

Comment:
Oops. Make this CPU-only because torch doesn't have `betainc` and we can't move data from GPU to CPU.